projects/plugins/crm/admin/settings/field-options.page.php

```diff
@@ -450,7 +460,7 @@ class="form-control" />
 
 								if ( empty( $zbsFunnelStr ) ) {
 									// } Defaults:
-									$zbsFunnelStr = 'Lead,Contacted,Customer,Upsell';
+									$zbsFunnelStr = 'Lead,Contacted,Customer,Upsell'; // phpcs:ignore WordPress.NamingConventions.ValidVariableName.VariableNotSnakeCase
```

Comment:
I'm guessing there used to be a change here that we reverted back?
Should we remove the PHPCS ignore statement since that would make this part of the code "untouched" and therefore not scanned by PHPCS anyway 😄

Reply:
Ironically, removing it now considers it a changed line :(

Reply:
Hahaha! Sadge 😂